src/app/(authenticated)/[username]/page.tsx

```diff
@@ -12,8 +16,46 @@ export default async function page({ params }: PageProps) {
   return (
     <Container>
       <div className="flex">
-        <Avatar avatarUrl={userResult.value.avatar}></Avatar>
-        <h1> {userResult.value.displayName}</h1>
+        <div className="w-1/4">
+          <Avatar avatarUrl={userResult.value.avatar} size={150}></Avatar>
+        </div>
+        <div className="flex w-9/12 flex-col">
+          <div className="flex">
+            <h1 className="font-medium text-lg w-1/2">
+              {userResult.value.displayName}
+            </h1>
+            <ThemeButton
+              type="button"
+              className="p-10 hover:bg-teal-500 font-bold">
+              Follow
+            </ThemeButton>
+          </div>
+          <div className="mt-5">
+            <ul className="flex justify-between w-full md:w-9/12">
```

Comment:
`<ul>` is a [block-level element](https://www.w3schools.com/html/html_blocks.asp) that takes full width of its container by default, so I don't think `w-full` is needed